### PR TITLE
Added MailButler to Email Utilities

### PR DESCRIPTION
**Pull request by highvoltag3**
MailButler is your personal assistant for efficient and productive work
with Apple Mail.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,6 +133,7 @@
 ### Email Utilities
 
 - [Airmail](http://airmailapp.com/) - Lightning fast email client designed for El Capitan, and a free Beta.
+- [MailButler](https://www.feingeist.io/mailbutler/) - MailButler is your personal assistant for efficient and productive work with Apple Mail. With its various functions, it simplifies, optimizes and streamlines working with emails in your daily life.
```
Comment (iCHAIT) [inline README.md]: i think the description is too long, just keep it till Apple Mail maybe?

**Comment by Derimagia** [comment]
Wow what a shame, the plugin looks decent but the pricing model is absurd. Definitely not awesome. 

:-1: in my books.

**Comment by herrbischoff** [comment]
:-1: as well. The monthly pricing model for this kind of functionality is just not too becoming for an awesome application.

File: README.md (hunk 1 above)
Comment (highvoltag3) [inline README.md]: :+1: yeah thought of the same myself at first.. I'll make the change.

**Comment by highvoltag3** [comment]
@Derimagia @herrbischoff I agree with both of you, but I think there's an issue with their wording or something. I've been using it since it came out and haven't been billed a thing (using the free tier plan they have) and so far I have not gotten over the 20ish actions they state.

**Comment by highvoltag3** [comment]
@iCHAIT changes made.

**Pull request closed without merging**